components/zcash_protocol/src/consensus.rs

```diff
@@ -641,7 +641,7 @@ impl TryFrom<u32> for BranchId {
             0xc2d6_d0b4 => Ok(BranchId::Nu5),
             0xc8e7_1055 => Ok(BranchId::Nu6),
             #[cfg(zcash_unstable = "nu7")]
-            0xffff_ffff => Ok(BranchId::Nu7),
+            0x77190ad8 => Ok(BranchId::Nu7),
```

Comment:
This value is not yet established by a deployment ZIP, and so MUST NOT be set to any non-0xffff_ffff value.
```suggestion
            0xffff_ffff => Ok(BranchId::Nu7),
```